samples/net/sockets/tls_configurations/prj.conf

```diff
@@ -0,0 +1,49 @@
+# Kernel options
+CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_POSIX_API=y
```

Comment:
I think so since I'm using POSIX calls like `socket()` (https://docs.zephyrproject.org/latest/doxygen/html/group__bsd__sockets.html#ga5693d19a0bdff45a5cb09227683d8631)